arm/Microsoft.CognitiveServices/accounts/deploy.bicep

```diff
@@ -70,23 +70,24 @@ param diagnosticEventHubAuthorizationRuleId string = ''
 @description('Optional. Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.')
 param diagnosticEventHubName string = ''
 
-@description('Conditional. Subdomain name used for token-based authentication. Required if \'networkAcls\' are set.')
+@description('Conditional. Subdomain name used for token-based authentication. Required if \'networkAcls\' are set (including private endpoints).')
```

Comment:
Not sure I understand what you mean with `(including private endpoints)`

Reply:
Private Endpoints are not part of the NetworkAcls, but network-related and also require a subdomain name. 

I guess I could also say `Required if \'networkAcls\' or \'privateEndpoints\' are set.`?

Reply:
Updated